racksdb/web/ui/src/views/DatacenterDetailsView.vue

```diff
@@ -7,15 +7,40 @@ SPDX-License-Identifier: GPL-3.0-or-later -->
 <script setup lang="ts">
 import { useHttp } from '@/plugins/http'
 import { useRacksDBAPI } from '@/composables/RacksDBAPI'
-import { ref, onMounted, watch } from 'vue'
-import SearchBar from '@/components/SearchBar.vue'
+import { ref, onMounted, watch, computed } from 'vue'
 import type { Ref } from 'vue'
 import type { Datacenter, DatacenterRoom } from '@/composables/RacksDBAPI'
+import { BarsArrowDownIcon, BarsArrowUpIcon, ChevronRightIcon } from '@heroicons/vue/24/outline'
+import { useRouter } from 'vue-router'
 
+const route = useRouter()
```

Comment:
The variable is not properly named here as `useRouter()` returns a router and not a route.